apps/builder/app/builder/features/navigator/navigator-tree.tsx

```diff
@@ -583,6 +584,11 @@ export const NavigatorTree = () => {
                       ? rawTheme.colors.foregroundReusable
                       : undefined,
                   },
+                  onKeyDown: (event) => {
+                    if ($textEditingInstanceSelector.get() !== undefined) {
+                      event.preventDefault();
+                    }
```

Comment:
Is it still necessary after removing focus on select?

Reply:
Pretty sure that yes, as it had intersection with key down during the editing